tests/Integration/Routing/ImplicitBindingTest.php

```diff
@@ -50,7 +67,11 @@ public function testPreviousUrlWithoutSession()
 
         $this->artisan('route:cache')->run();
 
-        $this->reloadApplicationWithCachedRoutes();
+        $this->reloadApplication();
+
+        $this->assertFilenameExists('bootstrap/cache/routes-v7.php');
+
+        $this->requireApplicationCachedRoutes();
```

Comment:
Split the method in case anyone just need to require cached routes on existing setup (without running `route:cache`)